### PR TITLE
Review

### PR DESCRIPTION
**Pull request by jsug9**
- Create a new branch for task.
- Add files from old project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  <h1 id="title">freeCodeCamp Survey Form</h1>
+  <p id="description">freeCodeCamp Survey Form</p>
+  <form id="survey-form">
+    <label id="name-label" for="name">Name</label>
+    <input type="text" id="name" placeholder="John Doe" required>
+    <label id="email-label" for="email">Email</label>
+    <input type="email" id="email" placeholder="email@email.com" required>
+    <label id="number-label" for="number">Age</label>
+    <input type="number" id="number" min="21" max="90" placeholder="Ex: 25">
```
Comment (jsug9) [inline index.html:15]: [Optional] In the long term, adding a class to certain objects like p and label could be useful for better customization.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <label><input name="prefer" value="data" type="checkbox">Data Visualization</label>
+    <label><input name="prefer" value="challenges" type="checkbox">Challenges</label>
+    <label><input name="prefer" value="open" type="checkbox">Open Source Community</label>
+    <label><input name="prefer" value="gitter" type="checkbox">Gitter help rooms</label>
+    <label><input name="prefer" value="videos" type="checkbox">Videos</label>
+    <label><input name="prefer" value="city" type="checkbox">City Meetups</label>
+    <label><input name="prefer" value="wiki" type="checkbox">Wiki</label>
+    <label><input name="prefer" value="forum" type="checkbox">Forum</label>
+    <label><input name="prefer" value="additonal" type="checkbox">Additional Courses</label><br>
+    <textarea></textarea>
```
Comment (jsug9) [inline index.html:39]: The text area should also have a name.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <h1 id="title">freeCodeCamp Survey Form</h1>
+  <p id="description">freeCodeCamp Survey Form</p>
+  <form id="survey-form">
```
Comment (jsug9) [inline index.html:9]: It'd be great if the form has a test action to check if it is submitting the information correctly.

**Review by jsug9 — COMMENTED** [review]
### Hello @jsug9,
You have done a great job so far!
- Excellent semantics. ✅
- Good Github Flow best practices. 👌

### Suggestions
There are some issues that you still need to work on, but you are almost there!
Check the comments below:
- You should add linters so they can help you keep your code as clean as possible.
- A gitignore file would be great to hide .DS_Store files, which are unnecessary to see on the repo.
- Consider adding css code to make your project look more appealing.

**Review by thecodechaser — APPROVED** [review]
Hi @jsug9 👋, 

Your project is complete!  There is nothing else to say other than Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**